src/mumble/GlobalShortcut_win.h

```diff
@@ -45,9 +45,17 @@ typedef QPair<GUID, DWORD> qpButton;
 
 struct InputDevice {
 	LPDIRECTINPUTDEVICE8 pDID;
+
+	QString name;
+
 	GUID guid;
 	QVariant vguid;
-	QString name;
+
+	GUID guidproduct;
```

Comment:
The big-picture plan is to log the pid/vid when a device is encountered, if available. Since not all devices have their pid/vid exposed via the guidproduct, we'd like to fall back to simply logging that.

That's why I moved them into the InputDevice structure.